integrationtests/tests/fixtures/mod.rs

```diff
@@ -832,6 +851,35 @@ impl FederationTest {
         }
     }
 
+    /// Removes the ecash nonces from the fed DB to simulate the fed losing track of what ecash
+    /// has already been spent
+    pub async fn clear_ecash_nonces(&self) {
+        for server in &self.servers {
+            block_on(async {
+                let svr = server.borrow_mut();
+                let mut dbtx = svr.database.begin_transaction().await;
+
+                {
+                    let mut module_dbtx =
+                        dbtx.with_module_prefix(LEGACY_HARDCODED_INSTANCE_ID_MINT);
```

Comment:
:(

Maybe we could make a `mint_module_id: ModuleInstanceId` in `FederationTest` and start using that to not proliferate this hardcoding that is supposed to go away?

Reply:
I'll do that as a separate refactor since it occurs in several of the tests